.gitignore

```diff
@@ -107,4 +107,7 @@ dist
 # TernJS port file
 .tern-port
 
-frontend/.vscode
+# .vscode
+.vscode/
```

Comment:
This is sufficient since it does exclude all directories named `.vscode` in all subdirectories. It seems there was a tasklist committed to the entry folder's `.vscode` subdir on purpose in order to make the generate api command more accessible from VS Code. Maybe we should not include it in the `.gitignore` list and just add the `backend/.vscode` dir in addition to `frontend`? We can also consider excluding everything except the tasks list in the entry dir.